### PR TITLE
seeding database with 100 rooms

### PR DESCRIPTION
**Pull request by danbergelt**

### CODE REVIEW

File: util/create_world.py [hunk 1]
```diff
@@ -1,47 +1,30 @@
...
+          roomTracker[(rm.x, rm.y)].connectRooms(roomTracker[rm.x - 1, rm.y], "w") 
+          roomTracker[(rm.x - 1, rm.y)].connectRooms(roomTracker[rm.x, rm.y], "e")
+        if rm.y % 2 != 0:   
+          if rm.x == 9 and rm.y > 0 and rm.y <= 9:
+            roomTracker[(rm.x, rm.y)].connectRooms(roomTracker[rm.x, rm.y - 1], "s")
+            roomTracker[(rm.x, rm.y - 1)].connectRooms(roomTracker[rm.x, rm.y], "n")
+        if rm.y > 0 and rm.y % 2 == 0:
+          if rm.x == 0:
+            roomTracker[(rm.x, rm.y)].connectRooms(roomTracker[rm.x, rm.y - 1], "s")
+            roomTracker[(rm.x, rm.y - 1)].connectRooms(roomTracker[rm.x, rm.y], "n")
```
Comment (brellin) [inline util/create_world.py:25]: We might not actually need all of these if statements. We'll see.

File: util/create_world.py [hunk 2]
```diff
@@ -1,47 +1,30 @@
...
+world.print_rooms()
+players = Player.objects.all()
 for p in players:
-  p.currentRoom=r_outside.id
-  p.save()
-
+    p.currentRoom = world.grid[0][0].id
+    p.save()
```
Comment (brellin) [inline util/create_world.py:30]: Good work

**Review by brellin — APPROVED** [review]

**Review by JCRN — APPROVED** [review]
looks good!

**Review by pdadlani — APPROVED** [review]
the algorithm makes sense to me. It's nice and clean too.

File: util/create_world.py (hunk 1 above)
Comment (pdadlani) [reply util/create_world.py:25]: ^ it works for now, but I was thinking the same thing

**Review by antilou86 — APPROVED** [review]

**Pull request merged**